src/index.html

```diff
@@ -4,9 +4,263 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap&subset=cyrillic-ext"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="head">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class ="navlist">
+        <ul class="nav">
+          <li class ="is-active">
+            <a href="#" class="link">Apple</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Samsung</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Смартфоны и телефоны</a>
+          </li>
+          <li class ="nav_point" data-qa="nav-hover">
+            <a href="#" class="link">Ноутбуки и компьютеры</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Гаджеты</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Планшеты</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Фото</a>
+          </li>
+          <li class ="nav_point">
+            <a href="#" class="link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container" data-qa="card">
+        <a href="#">
+          <img src="./images/imac.jpeg"
+            class="container__product"
+            alt="comp">
+        </a>
+        <h2 class="container__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <div class="container__number">Код товара: 195434</div>
+        <div class="container__stars">
+          <div class="container__stars container_4">
+            <div class="container_4_01"></div>
+            <div class="container_4_01"></div>
+            <div class="container_4_01"></div>
+            <div class="container_4_01"></div>
+            <div class="container_4_01"></div>
+          </div>
```

Comment:
```suggestion
          <div class="container__stars container_4">
            <div class="container__star"></div>
            <div class="container_4_01"></div>
            <div class="container_4_01"></div>
            <div class="container_4_01"></div>
            <div class="container_4_01"></div>
          </div>
```
Странно выглядит этот класс